vault/resource_consul_secret_backend.go

```diff
@@ -140,8 +142,18 @@ func consulSecretBackendCreate(d *schema.ResourceData, meta interface{}) error {
 	d.Partial(true)
 	log.Printf("[DEBUG] Mounting Consul backend at %q", path)
 
+	// If a token isn't provided and the Vault version is less than 1.11, fail before mounting the path in Vault.
+	bootstrapSupport, _, err := semver.GreaterThanOrEqual(ctx, client, consts.VaultVersion11)
```

Comment:
nit: `bootstrapSupport` reads like a verb, you may want to rename it to `bootstrapSupported` or `canBootstrap` or something generic like `useAPIVer1` or `isAPIVer1`

Reply:
Missed this one before pushing the extra change, thanks for catching.